utils/models.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+from django.db import models
+
+
+class BaseModel(models.Model):
+    class Meta:
+        abstract = True
+
+    def save(self, *args: Any, **kwargs: Any) -> None:  # pylint: disable=signature-differs
+        self.full_clean()
+        super(BaseModel, self).save(*args, **kwargs)
```

Comment:
I'm sure we need to update pylint because it also should be raised. Please not use python 2 inheritance method. Please use just `super().save(*args, **kwargs)`

Reply:
Done